src/care/ProfilePlugin.cpp

```diff
@@ -0,0 +1,66 @@
+#include "care/ProfilePlugin.h"
+
+#include "chai/ArrayManager.hpp"
```

Comment:
I think you can remove the include statements for ArrayManager.hpp, ExecutionSpaces.hpp, and CHAICallback.h.